packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [2.x.x]
```

Comment:
I think that, because the major version of dio, which is used in the packaged is changed from 4 to 5, the version of the package should be set to 3.0.0